data/json/monsters/mammal.json

```diff
@@ -1118,82 +1119,27 @@
     "melee_skill": 4,
     "melee_dice_sides": 2,
     "melee_damage": [ { "damage_type": "cut", "amount": 6 } ],
-    "special_attacks": [ [ "EAT_FOOD", 100 ] ],
     "vision_night": 4,
     "harvest": "dog_with_skull_leather",
     "reproduction": { "baby_monster": "mon_dog_bull_pup", "baby_count": 7, "baby_timer": 300 },
     "//": "7 puppies and 300-320 days per-litter for size medium canines",
-    "petfood": {
-      "food": [ "DOGFOOD" ],
-      "feed": "The %s seems to like you!  It lets you pat its head and seems friendly.",
-      "pet": "The %s happily wags its tail while you pat its head."
-    },
-    "flags": [
-      "ANIMAL",
-      "DOGFOOD",
-      "CANPLAY",
-      "CAN_BE_CULLED",
-      "PET_MOUNTABLE",
-      "HEARS",
-      "HIT_AND_RUN",
-      "KEENNOSE",
-      "PATH_AVOID_DANGER",
-      "SEES",
-      "SMELLS",
-      "WARM",
-      "CORNERED_FIGHTER",
-      "EATS"
-    ]
+    "delete": { "flags": [ "SWARMS", "GROUP_MORALE" ] }
```

Comment:
I wouldn't bother defining this. ``GROUP_MORALE`` fits dogs very well, at least.

Reply:
I agree, I thought it was fairly off that only one or two breeds of dogs retained their pack stuff so I went ahead and kept the GROUP_MORALE and SWARMS flags on each breed and attempted to swap the fear and anger fields by overwriting the anger entry on the base pups. I'm about to test it now and see if I can piss off a whole swarm of doggos. I also changed the zombify-into section like you mentioned and I'm going to test that as well.
Something that is odd to me though is I haven't been able to spot what exactly the SWARMS flag does in any of the docs I've Ctrl+F'd my way through looking for it. :/